### PR TITLE
[DEVOPS-278] [STAGING] Daedalus fixes for Windows

### PR DESCRIPTION
**Pull request by deepfire**
1. "fix" DEVOPS-290 -- we need proper NSIS in the build environment: https://issues.serokell.io/issue/DEVOPS-290
2. update Daedalus launch flags
3. since the `dns` library on Windows is broken, provide a topology file with _relay_ IP addresses hard-coded -- this binds the installer to `staging`
4. adapt `build-certiicates-win64.bat` to the changes that happened since it was introduced -- most importantly, make it use the `%APPDATA%` folder for intermediate/generated output.
5. fix the launch script constructor ending with a `^` -- this was causing a Haskell exception in `cardano-launcher`
6. redirect `report-server` output to `report-server.aws.iohkdev.io`
7. the Daedalus log directory no longer exists prior to `build-certificates-win64.bat` execution, so it has to be created

This still ends up blocked by https://issues.serokell.io/issue/DAEF-437

### CODE REVIEW

**Comment by domenkozar** [comment]
Great work :) Let me attempt to fix the CI. We also need to support macOS.

File: electron/remote/get-tls-config.js [hunk 1]
```diff
@@ -2,5 +2,5 @@ import path from 'path';
 import ClientApi from 'daedalus-client-api';
 
 export default function getTlsConfig() {
-  return ClientApi.tlsInit(path.join(__dirname, '../../tls/ca.crt'));
+  return ClientApi.tlsInit(path.join(__dirname, '../../tls/ca/ca.crt'));
```
Comment (akegalj) [inline electron/remote/get-tls-config.js]: I believe it should be `../../../../tls/ca/ca.crt` but lets try with this one.

Comment (DominikGuzei) [reply electron/remote/get-tls-config.js]: @akegalj im sorry i forgot to delete these files in `remote` folder, we are not using them anymore, so please ignore these paths.

Comment (darko-mijic) [reply electron/remote/get-tls-config.js]: I have removed unused files.

Comment (deepfire) [reply electron/remote/get-tls-config.js]: @akegalj, @DominikGuzei, aha, so we're _still_ going to refer to the CA cert relative to `%INSTALL_DIR%`?

So I need to fix `Launcher.hs` back?

Comment (akegalj) [reply electron/remote/get-tls-config.js]: @deepfire I wasn't paying attention to the discussion. 

@DominikGuzei and you agreed to have ca.cert at which dir ?

**Comment by deepfire** [comment]
As of https://github.com/input-output-hk/daedalus/pull/443/commits/cae2b78d8743deba54867bdc0546fbb45599ed1a, the FE/BE connection works for me on my local machine.

Waiting for CI..

**Comment by deepfire** [comment]
The Appveyor branch build actually succeeded, despite the status -- it's just the build cache update that failed.

Windows installer: https://ci.appveyor.com/api/buildjobs/lx8a73uq185eavrv/artifacts/installers%2Fdaedalus-win64-0.5.2730.0-installer.exe

File: scripts/build-installer-win64.bat [hunk 2]
```diff
@@ -24,6 +31,11 @@ set DLLS_URL=https://s3.eu-central-1.amazonaws.com/cardano-sl-testing/DLLs.zip
 @echo ..with LibreSSL version:    %LIBRESSL_VERSION%
 @echo .
 
+@if not [%FASTER_REBUILD%]==[] (@echo WARNING: FASTER_REBUILD set, skipping to frontend packaging
```
Comment (domenkozar) [inline scripts/build-installer-win64.bat]: FAST vs FASTER is ambigious, let's rather call them SKIP_TO_FRONTEND_BUILD and SKIP_TO_INSTALLER_BUILD

**Pull request closed without merging**